modules/Hoot/ui/modals/importDataset.js

```diff
@@ -57,6 +57,10 @@ export default class ImportDataset {
                 title: 'File (geonames, txt)',
                 value: 'GEONAMES'
             },
+            {
+                title: 'File (GeoPackage, gpkg)',
+                value: 'GPKG'
+            },
```

Comment:
I also wonder why we group `File (shp, zip, gdb.zip)` but we have separate entries for geojson, geonames, geopackage?